sandbox-web/src/main/java/com/exadel/sandbox/controllers/location_controller/CountryController.java

```diff
@@ -25,13 +25,15 @@ public CountryController(CountryService countryService) {
 
     @GetMapping(produces = {"application/json"}, path = "/country/{id}")
     public ResponseEntity<?> getCountryById(@PathVariable Long id) {
+
         final var countryById = countryService.getCountryById(id);
 
         return new ResponseEntity<>(countryById, HttpStatus.OK);
     }
 
     @GetMapping(produces = {"application/json"}, path = "/country")
     public ResponseEntity<?> getCountryByName(@RequestParam(name = "name", defaultValue = "") String name) {
```

Comment:
```
@RequestParam(name = "name", defaultValue = "")
```
Looks weird that default value is empty string.

Also Spring automatically map request param with the same name, it's not necessary to define name attribute:
```
@RequestParam String name
```